### PR TITLE
[FSDE-1100] Fixed minor toolbar component enhancements

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSDE-1100

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-96/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://0fc47af3ce1dd4c838b19befc326cabb500832c7-chi-pr.enterprise.ctl.io

**Comment by janibolkvadze** [comment]
@jllr Thoughts on adding Tooltips to the Icon Buttons of Toolbar?

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-96/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://3322625d8982175a9579e6e174ce7ec422eb1853-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-96/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ec808dfa3c39d100acefdf69094689c98609091f-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-96/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2db78ffe24aa1991e8047dea92163ada8ab7c143-chi-pr.enterprise.ctl.io

File: src/website/views/components/toolbar/_examples.pug [hunk 1]
```diff
@@ -215,10 +225,14 @@ p.-text
...
-          chi.drawer(document.getElementById('drawer-trigger-b1'));
-          chi.drawer(document.getElementById('drawer-trigger-b2'));
-        </script>
+      <script>
+        chi.drawer(document.getElementById('drawer-trigger-b1'));
+        chi.drawer(document.getElementById('drawer-trigger-b2'));
+        chi.tooltip(document.getElementById('button-b-filters'));
+        chi.tooltip(document.getElementById('button-b-download'));
+        chi.tooltip(document.getElementById('button-b-refresh'));
+        chi.tooltip(document.getElementById('button-b-column-customization'));
```
Comment (janibolkvadze) [inline src/website/views/components/toolbar/_examples.pug:234]:
```suggestion
        chi.tooltip(document.querySelectorAll('[data-tooltip]'));
```

File: src/website/views/components/toolbar/_examples.pug [hunk 2]
```diff
@@ -463,10 +488,14 @@ h3 Base with ribbon
...
-          chi.drawer(document.getElementById('drawer-trigger-br1'));
-          chi.drawer(document.getElementById('drawer-trigger-br2'));
-        </script>
+      <script>
+        chi.drawer(document.getElementById('drawer-trigger-br1'));
+        chi.drawer(document.getElementById('drawer-trigger-br2'));
+        chi.tooltip(document.getElementById('button-br-filters'));
+        chi.tooltip(document.getElementById('button-br-download'));
+        chi.tooltip(document.getElementById('button-br-refresh'));
+        chi.tooltip(document.getElementById('button-br-column-customization'));
```
Comment (janibolkvadze) [inline src/website/views/components/toolbar/_examples.pug:497]:
```suggestion
        chi.tooltip(document.querySelectorAll('[data-tooltip]'));
```

File: src/website/views/components/toolbar/_examples.pug [hunk 3]
```diff
@@ -687,10 +725,14 @@ h3 Base with ribbon
...
-            chi.drawer(document.getElementById('drawer-trigger-bp1'));
-            chi.drawer(document.getElementById('drawer-trigger-bp2'));
-          </script>
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-bp1'));
+          chi.drawer(document.getElementById('drawer-trigger-bp2'));
+          chi.tooltip(document.getElementById('button-p-filters'));
+          chi.tooltip(document.getElementById('button-p-download'));
+          chi.tooltip(document.getElementById('button-p-refresh'));
+          chi.tooltip(document.getElementById('button-p-column-customization'));
```
Comment (janibolkvadze) [inline src/website/views/components/toolbar/_examples.pug:734]:
```suggestion
          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
```

File: src/website/views/components/toolbar/_examples.pug [hunk 4]
```diff
@@ -939,10 +990,14 @@ h3 Base with ribbon
...
-            chi.drawer(document.getElementById('drawer-trigger-brp1'));
-            chi.drawer(document.getElementById('drawer-trigger-brp2'));
-          </script>
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-brp1'));
+          chi.drawer(document.getElementById('drawer-trigger-brp2'));
+          chi.tooltip(document.getElementById('button-pr-filters'));
+          chi.tooltip(document.getElementById('button-pr-download'));
+          chi.tooltip(document.getElementById('button-pr-refresh'));
+          chi.tooltip(document.getElementById('button-pr-column-customization'));
```
Comment (janibolkvadze) [inline src/website/views/components/toolbar/_examples.pug:999]:
```suggestion
          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
```

File: src/website/views/components/toolbar/_examples.pug [hunk 5]
```diff
@@ -1011,6 +1066,22 @@ script.
...
+      chi.tooltip(document.getElementById('button-br-refresh'));
+      chi.tooltip(document.getElementById('button-br-column-customization'));
+      chi.tooltip(document.getElementById('button-p-filters'));
+      chi.tooltip(document.getElementById('button-p-download'));
+      chi.tooltip(document.getElementById('button-p-refresh'));
+      chi.tooltip(document.getElementById('button-p-column-customization'));
+      chi.tooltip(document.getElementById('button-pr-filters'));
+      chi.tooltip(document.getElementById('button-pr-download'));
+      chi.tooltip(document.getElementById('button-pr-refresh'));
+      chi.tooltip(document.getElementById('button-pr-column-customization'));
```
Comment (janibolkvadze) [inline src/website/views/components/toolbar/_examples.pug:1084]:
```suggestion
      chi.tooltip(document.querySelectorAll('[data-tooltip]'));
```

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-96/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2edefdb5599ec86042c1aa2969193cf16bf114a6-chi-pr.enterprise.ctl.io

**Review by aryaroudi — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**